src/controller/AutoCommissioner.cpp

```diff
@@ -392,6 +392,13 @@ CommissioningStage AutoCommissioner::GetNextCommissioningStageInternal(Commissio
         }
         return CommissioningStage::kFindOperational;
     case CommissioningStage::kFindOperational:
+        if (mParams.GetIcdRegistrationStrategy() != IcdRegistrationStrategy::kIgnore)
```

Comment:
Why is this after FindOperational, exactly?

Reply:
no longer exists